plugins/imager.go

```diff
@@ -119,7 +119,7 @@ func NewImager() (imager *Imager) {
 					toMoosificate = toMoosificate[1 : len(toMoosificate)-1]
 				}
 
-				return fmt.Sprintf("http://www.moosificator.com/api/moose?image=%s", url.QueryEscape(toMoosificate))
+				return fmt.Sprintf("http://moosificator.herokuapp.com/api/moose?image=%s", url.QueryEscape(toMoosificate))
```

Comment:
I think we could use `https` instead of `http`, right? 🔐

Reply:
Hmm, is that because we previously didn't set up certs, but Heroku manages them for `*.herokuapp.com`? Anyway, good point... will change